change/@microsoft-teams-js-3261febf-9d87-4383-bfef-0a5b53d1f145.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Removed all of testing communicating functions tagged with `WithVersion` and now fully support sending api version tags for all of APIs",
```

```suggestion
  "comment": "Updated all capabilities to send API version telemetry",
```

Something like this will make more sense to external developers reading the patch notes

---

Thanks for your suggestion. Updated.